Explorer/Assets/DCL/InWorldCamera/PhotoDetail/PhotoDetailInfoView.cs

```diff
@@ -0,0 +1,23 @@
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace DCL.InWorldCamera.PhotoDetail
+{
+    public class PhotoDetailInfoView : MonoBehaviour
```

Comment:
Same as in previous comments. If everything is accessable by the controller, then the view loses its purpose. They should be all protected.